server/opts_test.go

```diff
@@ -1155,6 +1155,36 @@ func TestPanic(t *testing.T) {
 	}
 }
 
+func TestPingIntervalOld(t *testing.T) {
+	conf := createConfFile(t, []byte(`ping_interval: 5`))
+	defer os.Remove(conf)
+	opts := &Options{}
+	err := opts.ProcessConfigFile(conf)
+	if err == nil {
+		t.Fatalf("expected an error")
+	} else if err, ok := err.(*processConfigErr); !ok {
```

Comment:
Style: since t.Fatalf() will exit the routine, there is no need for the `else` statements. That is, you should remove all `else` and just have `if`.

Reply:
removed